### ISSUE TITLE
Writing FPL Playlist Not Working

### ISSUE THREAD

**Issue by nlogozzo** [Zeugma440/atldotnet#230]
According to [this](https://github.com/Zeugma440/atldotnet/blob/main/ATL/Playlist/PlaylistIOFactory.cs#L23), looks like FPL playlists are supported...however on writing the file we get an error saying `FPL writing not implemented`.

Is support planned for it soon? If not, I'd suggest removing this format from the list

**Comment by Zeugma440** [comment]
FPL is closed-source; there are no specs afaik. 

The project's welcome page does say support is read-only.

![image](https://github.com/Zeugma440/atldotnet/assets/29414549/95b49d4d-ef91-4a3a-8f2a-fa81eb8bf591)


I'm sorry but I can't go further on that one :(

**Comment by nlogozzo** [comment]
No worries...just cause we added playlist support to Tagger so I took the list of formats from there...but I'll just remove FPL since it's closed-source. Thanks!

**Comment by Zeugma440** [comment]
Should I implement a `Writable` boolean flag on `Format` to have that information usable by your app dynamically?

```cs
foreach (Format f in PlaylistIOFactory.GetInstance().getFormats())
{
    if (f.Readable)
    {
       // do stuff
    }
    if (f.Writable)
    {
       // do stuff
    }
}
```

**Comment by nlogozzo** [comment]
> Should I implement a `Writable` boolean flag on `Format` to have that information usable by your app dynamically?

Sure :)

**Comment by Zeugma440** [comment]
Will do for next release!

**Comment by nlogozzo** [comment]
> Will do for next release!

I'll reopen the issue to use to track :)

**Comment by Zeugma440** [comment]
Available on today's v5.08
